xcm/xcm-builder/src/weight.rs

```diff
@@ -114,6 +114,37 @@ where
 	}
 }
 
+/// Adapter for `UniversalWeigher` which can estimate weights for XCM message,
```

Comment:
```suggestion
/// Adapter for `UniversalWeigher` which can estimate weights for an XCM,
```

Reply:
Outside the scope of this PR, but since a lot of people get confused about XCM (as a pallet with some functions vs. an instruction set to write programs with), it may be clearer if we adopt some consistency in reference to messages. "XCM message" is a bit awkward because of "message message", "an XCM" isn't really clear (function, instruction). We could say "XCM instruction" when referencing a single instruction, or "XCM program" when (like here) we're dealing with an array of instructions. That is, we should generally use "XCM" as an adjective to modify {format, instruction, program}, not a noun.

Reply:
+1 on using "XCM program" for messages (collection of XCM instructions)